crates/protogen/src/sqlexec/physical_plan.rs

```diff
@@ -252,6 +252,8 @@ pub struct DropTablesExec {
     pub tbl_references: Vec<FullObjectReference>,
     #[prost(bool, tag = "3")]
     pub if_exists: bool,
+    #[prost(message, repeated, tag = "4")]
+    pub tbl_entries: Vec<TableEntry>,
```

Comment:
If we have table entries, do we need table references? Can we just use the info provided in entries?

Reply:
I originally wanted to go that route, but we don't have access to an entry resolver inside the execution context, only during planning.